### DISCUSSION TITLE
encore: command not found

### DISCUSSION THREAD

**Discussion by ohyoxo** [encoredev/encore#1121] [Help] [answered]
I have encountered the following problem following the deployment guide.
https://app.encore.dev/create-app

May I know how to solve it?
![image](https://github.com/encoredev/encore/assets/126952400/aa334c94-7586-458f-a39a-b5187873dcc7)

**Comment by eandre** [comment, marked as answer]
Hey @ohyoxo, you'll need to add `$HOME/.encore/bin` to your PATH environment variable. The instructions are listed on the screen you screenshotted (the two lines starting with `export`).
